query/src/storages/fuse/operations/mutation/block_filter.rs

```diff
@@ -0,0 +1,101 @@
+//  Copyright 2022 Datafuse Labs.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+use std::sync::Arc;
+
+use common_datablocks::DataBlock;
+use common_datavalues::DataSchemaRefExt;
+use common_exception::Result;
+use common_planners::Expression;
+
+use crate::pipelines::transforms::ExpressionExecutor;
+use crate::sessions::QueryContext;
+use crate::storages::fuse::meta::BlockMeta;
+use crate::storages::fuse::operations::mutation::mutations_collector::Deletion;
+use crate::storages::fuse::FuseTable;
+
+pub async fn delete_from_block(
```

Comment:
Why can't we reuse 'insert into overwrite table SELECT * FROM table WHERE delete_selector ' code? 
maybe we can support `INSERT INTO OVERWRITE_BLOCK table SELECT * FROM table WHERE delete_selector`.

Reply:
got it. please ignore